api/src/identity-access-management/application/token/token.controller.js

```diff
@@ -33,6 +34,9 @@ const createToken = async function (request, h, dependencies = { tokenService })
   const grantType = request.payload.grant_type;
 
   if (grantType === 'password') {
+    if (requestedApplication.isPixAdmin && !config.authentication.permitPixAdminLoginFromPassword) {
+      return h.response('PixAdminLoginFromPassword disabled. Use SSO authentication.').code(401).takeover();
+    }
```

Comment:
En terme de formatage, il serait plus logique d’ajouter une ligne sous ce bloc conditionnel car le code qui suit est sur une autre problématique : 

```suggestion
    if (requestedApplication.isPixAdmin && !config.authentication.permitPixAdminLoginFromPassword) {
      return h.response('PixAdminLoginFromPassword disabled. Use SSO authentication.').code(401).takeover();
    }

```